src/components/offline-indicator.tsx

```
@@ -0,0 +1,71 @@
+  // Don't render until mounted on client to avoid hydration mismatch
+  if (!mounted || !showIndicator) {
+    return null;
+  }
+
+  return (
+    <div
+      className={cn(
+        'fixed bottom-4 right-4 rounded-lg px-4 py-3 text-sm font-medium text-white shadow-lg transition-all',
+        isOnline
+          ? 'bg-green-500 animate-pulse'
+          : 'bg-red-500'
```

ESLint

[error] Replace `⏎··········?·'bg-green-500·animate-pulse'⏎·········` with `·?·'animate-pulse·bg-green-500'`

(prettier/prettier)